src/CsrfMiddleware.php

```diff
@@ -83,9 +91,14 @@ public function getHeaderName(): string
         return $this->headerName;
     }
 
+    public function getSafeMethods(): array
```

Comment:
Is it really needed? What case for this method?

Reply:
no cases indeed